src/test/java/com/amihaiemil/docker/RtContainerITCase.java

```diff
@@ -58,4 +62,80 @@ public void renamesContainer() throws Exception {
         );
         container.remove();
     }
+
+    /**
+     * {@link RtContainer} can start/stop Docker container it represents.
+     * @throws Exception If something goes wrong.
+     */
+    @Test
+    public void startStopContainer() throws Exception {
+        final Container container = new LocalDocker(
+            new File("/var/run/docker.sock")
+        ).containers().create("TestStart", this.containerJsonObject());
+        container.start();
+        MatcherAssert.assertThat(
+            container.inspect().getJsonObject("State").getBoolean("Running"),
```

Comment:
@bkuzmic Maybe it will be better if we extract  "Running" value to a constant field.

Reply:
@paulodamaso I've extracted this call to a separate method. It looks cleaner to me. What do you think?